crates/bevy_pbr/src/light.rs

```diff
@@ -822,25 +778,28 @@ pub(crate) fn assign_lights_to_clusters(
         lights.truncate(MAX_POINT_LIGHTS);
     }
 
-    let light_count = lights.len();
-    let mut global_lights_set = HashSet::with_capacity(light_count);
-    for (view_entity, view_transform, camera, frustum, config, mut clusters) in views.iter_mut() {
-        if matches!(config, ClusterConfig::None) {
+    for (view_entity, camera_transform, camera, frustum, config, clusters, mut visible_lights) in
+        views.iter_mut()
+    {
+        if matches!(config, ClusterConfig::None) && visible_lights.is_some() {
             commands.entity(view_entity).remove::<VisiblePointLights>();
             continue;
         }
 
-        let view_transform = view_transform.compute_matrix();
+        let clusters = clusters.into_inner();
+        let screen_size = camera.target.get_physical_size(&windows, &images);
+
+        clusters.aabbs.clear();
+        clusters.lights.clear();
+
+        let screen_size = screen_size.unwrap_or_default();
```

Comment:
yup! (this is the same approach used on `main`)

Reply:
I think `dimensions_for_screen_size` just below would produce somewhat odd results, albeit non-zero dimensions. I guess it's ok, but it feels like it could be better like falling back to a known-sensible configuration in nonsensical situations?

Reply:
I just made it gracefully handle zero screen sizes as outlined here: https://github.com/bevyengine/bevy/pull/4169#discussion_r824256927